### PR TITLE
LTD-5017 rename item list codes to denial cle

### PR DESCRIPTION
**Pull request by markj0hnst0n**
### Aim

Change pages that use denials to display the newly renamed variable

[LTD-5017](https://uktrade.atlassian.net/browse/LTD-5017)

Corresponding API PR [here](https://github.com/uktrade/lite-api/pull/2000)

[LTD-5017]: https://uktrade.atlassian.net/browse/LTD-5017?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: caseworker/templates/external_data/denial-detail.html [hunk 1]
```diff
@@ -61,7 +61,7 @@ <h1 class="govuk-heading-l">Denial details record</h1>
             </tr>
             <tr class="govuk-table__row app-table__row">
                 <th scope="row" class="govuk-table__header">Item List Codes</th>
```
Comment (depsiatwal) [inline caseworker/templates/external_data/denial-detail.html:63]: we probably also want to rename the header here

Comment (markj0hnst0n) [reply caseworker/templates/external_data/denial-detail.html:63]: It's actually done in [this](https://github.com/uktrade/lite-frontend/pull/1970) other PR as it was a separate story

File: caseworker/templates/case/denial-for-case.html [hunk 2]
```diff
@@ -80,7 +80,7 @@ <h2 class="govuk-heading-l govuk-!-margin-top-6 govuk-!-margin-bottom-3">
                                 <th scope="col" class="govuk-table__header">Name</th>
                                 <th scope="col" class="govuk-table__header">Address</th>
                                 <th scope="col" class="govuk-table__header">Country</th>
-                                <th scope="col" class="govuk-table__header">Item list codes</th>
+                                <th scope="col" class="govuk-table__header">Denial control list entry</th>
```
Comment (depsiatwal) [inline caseworker/templates/case/denial-for-case.html:83]: is it entry and entries ?

Comment (markj0hnst0n) [reply caseworker/templates/case/denial-for-case.html:83]: Entry as per the instructions on the story
image

**Review by depsiatwal — APPROVED** [review]
lgtm

**Review by Tllew — APPROVED** [review]
Looks good to me, checked frontend and I wasn't able to find any other instances of item_list_codes

**Pull request merged**